mathml/global_attributes.json

```diff
@@ -280,6 +277,41 @@
             "standard_track": true,
             "deprecated": true
           }
+        },
+        "small_normal_big": {
```

Comment:
Since this groups several features together and isn't a web platform feature name, this identifier needs a description. 
```suggestion
        "small_normal_big": {
          "description": "<code>small</code>, <code>normal</code>, <code>big</code> values",
```

Reply:
Thanks, I wanted to check the doc about that but forgot.

Reply:
mmh, not sure it's working... let me check...

Reply:
OK Done now.